src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -370,6 +370,11 @@ public string DefaultPythonEngine
         [XmlIgnore]
         public bool NamespacesToExcludeFromLibrarySpecified { get; set; }
 
+        /// <summary>
+        /// Settings that apply to view extensions.
+        /// </summary>
+        public List<ViewExtensionSettings> ViewExtensionSettings { get; set; }
```

Comment:
The setting here is good design. I also wonder when we about do make internal window like Python editor dockable, do we apply same setting for them or use a list of different but similar setting? Or we make this setting more generic? Let me know what you think. I think it might be something we want to consider

Reply:
I don't think this can be reused just like that. `ViewExtensionSettings` by itself contains view extension specific properties like `UniqueId` and `Name` that don't apply to script editors. Still, a lot of things in this contribution can be reused.

Reply:
@mmisol Yes, and thinking more on this. We may not need to support remembering the positions of python editor at all. Will take a look at the rest of code